src/main/java/org/sonarqube/gradle/AndroidUtils.java

```diff
@@ -101,6 +108,49 @@ private static void configureForAndroid(Project project, BaseVariant variant, Ma
     }
   }
 
+  private static void configureTestReports(Project project, BaseVariant variant, Map<String, Object> map) {
+    if (!(variant instanceof TestedVariant)) {
+      return;
+    }
+
+    if (GradleVersion.current().compareTo(GradleVersion.version("6.0")) < 0) {
```

Comment:
Should it depend on android gradle plugin version instead (com.android.tools.build:gradle), since TestedVariant is part of this package?